graphql_api/types/bundle_analysis/__init__.py

```diff
@@ -0,0 +1,21 @@
+from graphql_api.helpers.ariadne import ariadne_load_local_graphql
```

Comment:
More boilerplate imports and refactored the folder structure to be:
```
bundle_analysis/
  base.py
  report.py
  comparison.py
```
before it was
```
bundle_analysis_comparison/
  comparison.py
bundle_analysis_report/
  report.py
```